src/tasks/NetTraceToMibcConverterTask/NetTraceToMibcConverter.cs

```diff
@@ -0,0 +1,138 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Collections.Concurrent;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Text.Json.Serialization;
+
+public class NetTraceToMibcConverter : Microsoft.Build.Utilities.Task
```

Comment:
Should this be a ToolTask?

https://docs.microsoft.com/en-us/dotnet/api/microsoft.build.utilities.tooltask?view=msbuild-17-netcore

Reply:
It does seem rather heavyweight. It also feels like we should give the aot compiler task the same treatment. My first thought is to stay with what we have, but I can be convinced otherwise.

Reply:
With MonoAOTCompiler we do lot of custom stuff for running the tool - like running in parallel. ToolTask is for a single execution.

Reply:
OK, it wasn't as bad as I thought.